### ISSUE TITLE
Unable to build JSDuck gem

### ISSUE THREAD

**Issue by f1ames** [ckeditor/jsduck#4]
Following the build instructions:

> 6. `> rake configure`

![image](https://user-images.githubusercontent.com/1061942/29357821-30fef226-8279-11e7-9454-ece04ebc1525.png)

The link to `extjs` library used by `Rakefile` is outdated. Looks like Sencha does not provide the download anymore. I was able to overcome the issue using some 3rd party url like:

```
# Download ExtJS into template/extjs
task :get_extjs do
  system "curl -o template/extjs.zip https://ext4all.com/ext/download/ext-4.1.1-gpl.zip"
  system "unzip template/extjs.zip -d template/"
  system "rm -rf template/extjs"
  system "mv template/extjs-4.1.1 template/extjs"
  system "rm template/extjs.zip"
end
```

Then this step seems to be working fine properly downloading and extracting `ExtJS` library, however next step fails:

> 7. `> rake ext4`

![image](https://user-images.githubusercontent.com/1061942/29358221-854c4ca6-827a-11e7-8f64-dff15e911173.png)

**Comment by f1ames** [comment]
Seems like an issue with dependencies, newer `compass` releases does not include `blueprint`.

**Comment by f1ames** [comment]
SDK link https://github.com/Shereef/Sencha-Touch-2.

**Comment by f1ames** [comment]
I was able to run `rake ext4` and `rake gem` with following config (important are `compass 0.12.2` and `sass 3.1.1`):

![image](https://user-images.githubusercontent.com/1061942/29361664-ec4f70dc-8288-11e7-80b7-d41debad0f9a.png)

However, generating gem still produces some errors:

![image](https://user-images.githubusercontent.com/1061942/29361687-fe299f44-8288-11e7-8adb-6bf98314b683.png)

**Comment by f1ames** [comment]
It seems `sencha` command was not available, after restarting terminal it looks better:

![image](https://user-images.githubusercontent.com/1061942/29362075-ad1fbd02-828a-11e7-9f44-6ee98752cca6.png)

I am only concerned now about `cp: template/extjs/extjs: directory causes a cycle`, the rest looks like configuration/deps warnings which can be easily solved.

**Comment by f1ames** [comment]
Closed in #5.
